flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -629,15 +636,17 @@ function wait_oper_metric_num_in_records {
     done
 }
 
+function wait_num_of_occurence_in_logs_mesos() {
+    local text=$1
+    local number=$2
+    wait_num_of_occurence_in_logs $1 $2 "mesos-appmaster"
+}
+
 function wait_num_of_occurence_in_logs {
     local text=$1
     local number=$2
     local logs
-    if [ -z "$3" ]; then
-        logs="standalonesession"
-    else
-        logs="$3"
-    fi
+    local logs=${3-standalonesession}
```

Comment:
Very nice simplification.

nit: So far we have always used the `${3:-standalonesession}` syntax for default values in this project.